src/Services/Terminal/Terminal.js

```diff
@@ -95,6 +103,9 @@ module.exports = function (settings) {
       .then(processResponse)
       .then(
         (response) => {
+          if (debug) {
```

Comment:
In serverless setup for TCP-air we have debug more set to 1. I wanted to keep this output for debug level 1, as in debug level 2 we have all the XML output. Let's keep it like that. I'd like to have commands output for any debug level